ocpp-v1_6/src/main/java/eu/chargetime/ocpp/JSONClient.java

```diff
@@ -67,8 +67,9 @@ public JSONClient(ClientCoreProfile coreProfile, String identity) {
         featureRepository.addFeatureProfile(coreProfile);
     }
 
-    public void enableWSS(SSLContext sslContext) throws IOException {
-        transmitter.enableWSS(sslContext);
+    public JSONClient enableWSS(WssSocketBuilder wssSocketBuilder) throws IOException {
```

Comment:
This is an exposed API, which means that changes will break peoples code. I suggest that we keep the original one and add this as an overload.

Reply:
Good point. Updated class to be backward compatible.